src/Controllers/MagicLink/AppController.php

```diff
@@ -267,4 +234,41 @@ public function reset_apps( Request $request )
 
         return response( [ 'message' => 'App order updated' ] );
     }//end reset_apps()
+
+    public function get_apps( Request $request ) {
```

Comment:
You don't seem to be using the request parameter, so you are safe to omit it.

Reply:
@incraigulous  "The change request has been updated."